elasticsearch-plugins/resources/elasticsearch-plugin-batch.json

```diff
@@ -0,0 +1,3 @@
+{
+  "parents" : [ "cdap-etl-batch[0,100]" ]
```

Comment:
I agree we should probably change it, since there is only 1 version of this artifact available right now. Needs to be changed everywhere? guides, et all?